packages/liquidsoap/liquidsoap.2.0.1/opam

```diff
@@ -89,13 +83,9 @@ conflicts: [
   "faad" {< "0.5.0"}
   "fdkaac" {< "0.3.1"}
   "ffmpeg" {< "1.1.0"}
+  "ffmpeg" {> "1.1.0"}
   "ffmpeg-avutil" {< "1.1.0"}
-  "ffmpeg-av" {< "1.1.0"}
-  "ffmpeg-avcodec" {< "1.1.0"}
-  "ffmpeg-avdevice" {< "1.1.0"}
-  "ffmpeg-avfilter" {< "1.1.0"}
-  "ffmpeg-swresample" {< "1.1.0"}
-  "ffmpeg-swscale" {< "1.1.0"}
```

Comment:
Couldn’t they not have `{>= "1.1.1"}` like the two other ones above?

Reply:
I'm not sure if there's a great solution. As I said, I believe the correct solution is to let this one in and then add liquidsoap `2.0.2`, which fixes the underlying issue and would be the default pick for the `2.0.x` branch.

Reply:
But it changes the behaviour of this package quite heavily and enables race-condition during the compilation. What is so bad about adding `{>= "1.1.1"} ?

Reply:
I'm happy to implement either changes. If I add these back, the CI revdeps will start failing again, which is what triggered those changes in the first place.

Reply:
@kit-ty-kate Any preference here?

Reply:
I’d rather keep the depopts as they are currently and add a new conflict for each

Reply:
@kit-ty-kate You got it. Done! Thanks for your time with this.